SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -0,0 +1,108 @@
+%global debug_package %{nil}
+Summary:        NVIDIA container runtime hook
+Name:           nvidia-container-toolkit
+Version:        1.4.2
+Release:        2%{?dist}
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/nvidia-container-runtime
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        nvidia-container-toolkit-1.4.2.tar.gz
+Source1:        vendor.tar.gz
```

Comment:
Please remove the tarball from the repo - we'll need to add it to our sources repository the same way as `Source0`.